libs/datasets/timeseries.py

```diff
@@ -682,17 +682,25 @@ def to_csv(self, path: pathlib.Path):
             provenance_path = str(path).replace(".csv", "-provenance.csv")
             self.provenance.sort_index().rename(PdFields.PROVENANCE).to_csv(provenance_path)
 
-    def join_columns(
-        self, other: "MultiRegionTimeseriesDataset", replace: bool = False
-    ) -> "MultiRegionTimeseriesDataset":
+    def drop_column_if_present(self, column: str) -> "MultiRegionTimeseriesDataset":
```

Comment:
A unit test that exercises this would be nice.

Reply:
Yeah. I will try to do that tomorrow.  Sorry!